recipes/asciimatics/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "asciimatics" %}
+{% set version = "1.9.0" %}
+{% set sha256 = "4fa4ab85f414228cf7986d2ffde68f53685c45125ac0fd889dfac8069fd4d339" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - pyfiglet >=0.7.2
+    - Pillow >= 2.7.0
+    - future
+  run:
+    - python
+
+test:
+  imports:
+    - asciimatics 
+    - asciimatics.tests
+
```

Comment:
If possible run the unit tests to verify the dependencies. 

This is optional but it helps during the auto-tick bot PRs